vault/data_source_kubernetes_auth_backend_config_test.go

```diff
@@ -52,14 +52,15 @@ func TestAccKubernetesAuthBackendConfigDataSource_basic(t *testing.T) {
 func TestAccKubernetesAuthBackendConfigDataSource_full(t *testing.T) {
 	backend := acctest.RandomWithPrefix("kubernetes")
 	jwt := kubernetesJWT
+	issuer := "kubernetes/serviceaccount"
```

Comment:
Would it be possible to change the test string to something different than the default, and then check that it's read correctly? And then to have a second test where the issue isn't included, and we see it comes back as the default?

The reason I ask is because this test is a great start, but unclear to me. If we actively set the test value to the default, how do we know we're getting the actively set value versus the default?

I may be reading this incorrectly, pardon me if I am.

Reply:
there is another test that actually tests that issuer is changed https://github.com/terraform-providers/terraform-provider-vault/pull/601/files#diff-da563ce4f8d9e411cfe23aa99dfb0005R225 but I agree there must be a test with issuer not set and check if it comes back as default.